src/System.Slices/tests/UsageScenarioTests.cs

```diff
@@ -33,6 +33,7 @@ public MyByte(byte value)
         [InlineData(new byte[] { 0, 1, 2 })]
         [InlineData(new byte[] { 0, 1, 2, 3 })]
         [InlineData(new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 })]
+        [CLSCompliant(false)]
```

Comment:
The weird fact is that this whole assembly has CLSCompliant(false) and compiler still complains about it. I guess this is ok workaround for now